pkg/opcfg/config.go

```diff
@@ -378,10 +378,15 @@ func getStackTrace() zap.Option {
 
 // GetVdbMaxBackoffDuration returns maximum backoff requeue duration in milliseconds for vdb controller
 func GetVdbMaxBackoffDuration() int {
-	duration := lookupIntEnvVar("VDB_MAX_BACKOFF_DURATION", envCanNotExist)
-	if duration == 0 {
+	const envName = "VDB_MAX_BACKOFF_DURATION"
+	duration := lookupStringEnvVar(envName, envCanNotExist)
```

Comment:
Why not use `lookupIntEnvVar`? If the issue is the default value, maybe we can change this to `lookupStringEnvVar(envName string, mustExist bool, defaultValue int)`?

Reply:
`lookupIntEnvVar` returns 0 when it cannot find the env var which cannot differentiate the case with user input 0. I don't want to update `lookupIntEnvVar` too much since it's called in many other places.